### PR TITLE
[zh] Translate pod-template-v1.md

### PR DESCRIPTION
**Pull request by tengqm**
Author: Jin Li <qlijin@gmail.com>
Co-Author: Qiming Teng <tengqm@outlook.com>
Replaces: #34319 
Related: #34733

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+201 (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplate" >}}">PodTemplate</a>): Created
+
+401: Unauthorized
+
+<!--
+### `patch` partially update the specified PodTemplate
+
+#### HTTP Request
+-->
+`patch` 部分更新指定的 PodTemplate
```
Comment (ydFu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:516]:
```suggestion
### `patch` 部分更新指定的 PodTemplate
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+- **apiVersion**: v1
+
+- **kind**: PodTemplate
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准的对象元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (ydFu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:46]:
```suggestion
  标准的对象元数据。更多信息：
  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

**Review by ydFu — APPROVED** [review]
/lgtm

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 3]
```diff
@@ -0,0 +1,779 @@
...
+- **namespace** (*in path*): string, required
+-->
+- **namespace** （**路径参数**）：string，必需
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+<!--
+- **body**: <a href="{{< ref "../workload-resources/pod-template-v1#PodTemplate" >}}">PodTemplate</a>, required
+-->
+- **body**: <a href="{{< ref "../workload-resources/pod-template-v1#PodTemplate" >}}">PodTemplate</a>,必需
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:393]:
```suggestion
- **body**: <a href="{{< ref "../workload-resources/pod-template-v1#PodTemplate" >}}">PodTemplate</a>，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 4]
```diff
@@ -0,0 +1,779 @@
...
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:471]:
```suggestion
- **body**: <a href="{{< ref "../workload-resources/pod-template-v1#PodTemplate" >}}">PodTemplate</a>，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 5]
```diff
@@ -0,0 +1,779 @@
...
+- **namespace** (*in path*): string, required
+-->
+- **namespace** （**路径参数**）：string，必需
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+<!--
+- **body**: <a href="{{< ref "../common-definitions/patch#Patch" >}}">Patch</a>, required
+-->
+- **body**: <a href="{{< ref "../common-definitions/patch#Patch" >}}">Patch</a>,必需
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:547]:
```suggestion
- **body**: <a href="{{< ref "../common-definitions/patch#Patch" >}}">Patch</a>，必需
```

**Review by Sea-n — COMMENTED** [review]
Some nitpicks, still looks good to me.

**Comment by mengjiao-liu** [comment]
/approve
Thanks!

**Pull request merged**